README.md

```diff
@@ -8,6 +8,15 @@ There are two ways to develop this project:
 - Frontend development only, using the AWS integration environment for everything else. This is the default
 - Full stack local development, using a local dev copy of the API, Keycloak, etc
 
+
+- When moving between full local and frontend only development you should clear your browsers' cookies and local storage as the browser can cache the local Cognito ID, which is incompatible with non-local Cognito. An example of the error you will see if this happens is below:
```

Comment:
Sorry, just noticed that this looks like it's in the list of ways to develop the project. Maybe just remove the `-` from the start of this comment?

Reply:
Sure will do! I wasn't 100% sure on where to put this bit but I thought somewhere near this section was a safe bet.